pegasus/sites.v3/csedweek.org/views/map.haml

```diff
@@ -8,7 +8,7 @@
         = I18n.t(:csedweek_map_signup)
 
   .row
-    .col-xs-10.col-sm-12
+    .col-sm-12
```

Comment:
are we getting rid of the gutter on the right?  this was done explicitly so you couldn't get into a state where you were only scrolling the map but couldn't scroll the page.